etor/src/main/resources/openapi_etor.yaml

```diff
@@ -105,6 +105,38 @@ paths:
             application/json:
             schema:
               $ref: '#/components/schemas/NotFoundError'
+  /v1/etor/metadata/summary/{sender}:
+    get:
+      summary: Returns all the order and their status for particular hospital
+      parameters:
+        - name: sender
+          description: A valid sender for the partner metadata
+          required: true
+          schema:
+            type: string
+      security:
+        - bearerAuthorization: [ ]
+      responses:
+        '200':
+          description: orders and their status returned successfully
+          content:
+            application/json:
+            schema:
+              $ref: 'https://github.com/LinuxForHealth/FHIR/blob/main/fhir-openapi/src/main/webapp/META-INF/openapi.json?raw=true#/components/schemas/OperationOutcome'
+        '401':
+          description: Authentication failed due to invalid token or unknown organization
+          content:
+            application/json:
+            schema:
+              $ref: '#/components/schemas/UnauthorizedRequestError'
+        '404':
```

Comment:
Is the endpoint returning 404? I didn't find it. If it's not, it would be a good idea to add the condition to return 404 in case nothing is found

Reply:
I think this should be 500 not 404

Reply:
500 error codes are for unexpected internal errors. Not finding any metadata should be an expected outcome